app/Helpers/render/game.php

```diff
@@ -270,29 +270,29 @@ function RenderGameAlts(array $gameAlts, ?string $headerText = null): void
             $consoleName = null;
         }
 
-        $gameData = [
-            'ID' => $nextGame['gameIDAlt'],
-            'Title' => $nextGame['Title'],
-            'ImageIcon' => $nextGame['ImageIcon'],
-            'ConsoleName' => $consoleName,
+        $rowData = [
+            'gameId' => $nextGame['gameIDAlt'],
+            'gameTitle' => $nextGame['Title'],
+            'gameImageIcon' => $nextGame['ImageIcon'],
+            'consoleName' => $consoleName,
+            'totalPoints' => $points,
+            'totalRetroPoints' => $totalTP,
+            'isFullyFeaturedGame' => $isFullyFeaturedGame,
         ];
 
-        echo "<td>";
-        echo gameAvatar($gameData, label: false);
-        echo "</td>";
-
-        echo "<td style='width: 100%' " . ($isFullyFeaturedGame ? '' : 'colspan="2"') . ">";
-        echo gameAvatar($gameData, icon: false);
-        echo "</td>";
-
-        if ($isFullyFeaturedGame) {
-            echo "<td>";
-            echo "<span class='whitespace-nowrap'>$points points</span><span class='TrueRatio'> ($totalTP)</span>";
-            echo "</td>";
-        }
-
-        echo "</tr>";
+        echo Blade::render('
+            <x-game.similar-game-table-row
+                :gameId="$gameId"
+                :gameTitle="$gameTitle"
+                :gameImageIcon="$gameImageIcon"
+                :consoleName="$consoleName"
+                :totalPoints="$totalPoints"
+                :totalRetroPoints="$totalRetroPoints"
+                :isFullyFeaturedGame="$isFullyFeaturedGame"
+            />
+        ', $rowData);
```

Comment:
Allows to pass the `$rowData` array directly as the local variable isn't used anywhere else

Reply:
Sorry, I'm not sure I'm following.

Reply:
Could be passed directly like this - not very important though :)
```php
echo Blade::render('
    <x-game.similar-game-table-row
        :gameId="$gameId"
        :gameTitle="$gameTitle"
        :gameImageIcon="$gameImageIcon"
        :consoleName="$consoleName"
        :totalPoints="$totalPoints"
        :totalRetroPoints="$totalRetroPoints"
        :isFullyFeaturedGame="$isFullyFeaturedGame"
    />
', [
    'gameId' => $nextGame['gameIDAlt'],
    'gameTitle' => $nextGame['Title'],
    'gameImageIcon' => $nextGame['ImageIcon'],
    'consoleName' => $consoleName,
    'totalPoints' => $points,
    'totalRetroPoints' => $totalTP,
    'isFullyFeaturedGame' => $isFullyFeaturedGame,
]);
```

Reply:
No worries at all, addressed in https://github.com/RetroAchievements/RAWeb/pull/1690/commits/3909b0522d92c54474d3cf290cdf48e53e3717ad.